docs/README.md

```diff
@@ -61,7 +61,7 @@ There are several optional parameters that could be passed into `CreateVolumeReq
 
 2. Enable the flag `--allow-privileged=true` in the manifest entries of kubelet and kube-apiserver.
 
-3. Add `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,VolumeSnapshotDataSource=true` in the manifest entries of kubelet and kube-apiserver. This is required to enable topology support of EBS volumes in Kubernetes and restoring volumes from snapshots.
+3. Add `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true` in the manifest entries for `kube-apiserver` and `kube-controller-manager`; and `--feature-gates=CSINodeInfo=true` in the `kubelets`. This is required to enable topology support of EBS volumes in Kubernetes, using `volumeMode: Block` and restoring volumes from snapshots.
```

Comment:
Don't think `kube-controller-manager` needs those feature flags. I check `VolumeSnapshotDataSource`, it is for apiserver as well.

However, kubelet should have `CSINodeInfo`, `CSIDriverRegistry` and `CSIBlockVolume`. Haven't tested `VolumeSnapshotDataSource` on kubelet though